.circleci/config.yml

```
@@ -41,6 +41,8 @@ workflows:
                 root: testapp/build/outputs
                 paths:
                   - mapping/
+            - run: |
+                echo $(grep "VERSION_NAME" testapp/build/generated/source/buildConfig/staging/com/rakuten/tech/mobile/miniapp/testapp/BuildConfig.java | awk '{print $7}' | tr -d \''";\') > ./testapp/version-name.txt
```

I think your `tr` command is broken, on the end, here is one I tested
```suggestion
                echo $(grep "VERSION_NAME" testapp/build/generated/source/buildConfig/staging/com/rakuten/tech/mobile/miniapp/testapp/BuildConfig.java | awk '{print $7}' | tr -d '";') > ./testapp/version-name.txt
```

---

but, I think the previous one was printing the expected value correctly.